RadixWallet/Core/SharedModels/Entities/EntitiesControlledByFactorSource.swift

```diff
@@ -24,6 +24,55 @@ public struct EntitiesControlledByFactorSource: Sendable, Hashable, Identifiable
 }
 
 extension EntitiesControlledByFactorSource {
+	public struct PerCurve: Equatable, Sendable {
+		public struct ID: Sendable, Hashable {
+			public let factorSourceID: FactorSource.ID.FromHash
+			public let isOlympia: Bool
+		}
+
+		public let id: ID
+		public let accounts: NonEmpty<IdentifiedArrayOf<Profile.Network.Account>>
```

Comment:
Worth considering a typealias for `NonEmpty<IdentifiedArrayOf<Profile.Network.Account>>`?